Improve grammar (#8750)
TheOdinProject/curriculum: web_development_101/the_basics/introduction_to_web_development.md

Web development could be a good profession for you if you like solving logical problems, building useful things, and experimenting with new technologies.
Web developers are in high demand, generally have a good work/life balance, and command comfortable salaries. Google your specific location to get a better sense of your local web development job opportunities.

For more details, Wikipedia describes the breadth of the web development profession in their [entry on web design](https://en.wikipedia.org/wiki/Web_design).

### Types of Web Developers

Earlier, we mentioned that web development work could be in the front end, the back end, or the full stack. What exactly do these terms mean?

The **front end** is the stuff you see on the website in your browser, including the presentation of content and user interface elements like the navigation bar.
Front-end developers use HTML, CSS, JavaScript, and their relevant frameworks to ensure that content is presented effectively and that users have an excellent experience.

> The **back end** refers to the guts of the application, which live on the server. The back end manipulates data to ensure that the front end has what it needs, which can become very complicated as things scale.

Reviewer: ```suggestion
The **back end** refers to the guts of the application, which live on the server. The back end stores and serves program data to ensure that the front end has what it needs. This can become very complicated when a website has millions of users.
```

Back-end developers use programming languages like Java, Python, and Ruby to work with data.

**Full-stack developers** are comfortable working with both the front and back ends. In The Odin Project, we focus on teaching you full-stack development, covering all aspects of web development.

For more detail, Udacity has a great blog post on this topic: [3 Web Dev Careers Decoded: Front-End vs Back-End vs Full Stack](http://blog.udacity.com/2014/12/front-end-vs-back-end-vs-full-stack-web-developers.html)

### Types of Careers

Now that you know about the different types of web developers, let's cover what we mentioned earlier about the different types of clients and employers you may work with.

> Large tech companies, such as Google, Facebook, and Amazon, have very stringent hiring requirements, but they offer excellent pay, benefits, and opportunities.

Reviewer: Love "stringent" here 😍

Reviewer 2: That's a very long sentence with a lot of commas. I think it needs to be broken up and structured better?

### Motivation

> Learning to code is incredibly rewarding but can also be difficult and frustrating.

Reviewer: ```suggestion
Learning to code is incredibly rewarding, but can also be difficult and frustrating.
```

Author: @mindovermiles262 The comma that you've suggested to add in here is actually incorrect. It's an example of a compound predicate, where the subject ("Learning to code") takes two verbs ("is" and "can be"). In these situations, you don't want to put a comma between the subject and the verb. (For reference, see Chicago Manual of Style 6.29.) Alternatively, if you'd prefer to see a comma in this sentence, it could be changed to the following (adding in a second subject ("it") to make a compound sentence instead of a compound predicate):
"Learning to code is incredibly rewarding, but it can also be difficult and frustrating."

The strongest assets you can have as a student are a desire to build, a problem-solving mind, and persistence in the face of setbacks.

The web development industry has a long history of successful developers with varying backgrounds, so people tend to care more about what you've actually **built** than how you got there.

> So, you need to build. Build and build and build and keep on building as much as you can. Build projects to try out new technologies, and build things that you need.

Reviewer: ```suggestion
```

Reviewer: Let's remove this paragrph

Author: Happy to do so! I wasn't a huge fan of that one. :)

> Once you start to get comfortable with the tools, start hacking on open source projects [(like The Odin project itself)](/contributing). The more you practice, the more you will learn about what you can do, and the closer you will get to being hireable.

Reviewer: ```suggestion
Once you start to feel comfortable with the tools, start hacking on open source projects [(like The Odin project itself)](/contributing). The more you contribute, the more you will learn about what you can do, and the closer you will get to being hireable. These projects will also look great on your resume!
```

Read [this comprehensive blog post from Happy Bear Software](https://www.happybearsoftware.com/how-to-get-a-programmer-job) about the journey to getting hired as a brief introduction to what you will face ahead.

### Why Odin?

I want you to know that this will not be easy.

There are plenty of other online curriculums for beginners, but they are often taught in an extremely isolated and controlled environment and cover only a specific topic.

The Odin Project takes a realistic view of what you need to know and has you set up and work in your own environment, much like what you'll be doing when you get a job.

It acknowledges that you need a wide variety of skillsets and languages to reach an employable level.

> The Odin Project is constantly evolving because of people like you who get further along in the curriculum and pay it forward by incrementally improving our content over time.

Reviewer: Because of people like you @leila-alderman 😄

Author: Thanks! Happy to help out!

### Conclusion

Hopefully, you've gained a better idea of what a web developer actually does and what your life might look like if you decided to take it on as a career. This has only been a teaser into the world of web development.

> In this course, "Web Development 101," you'll take a journey through the entire spectrum of topics that you will eventually need to know. This course jumps around to a variety of topics that you may be totally unfamiliar with, giving you a small taste of each and then moving on.

Reviewer: ```suggestion
In this course, "Web Development 101," you'll take a journey through the entire spectrum of topics that you will eventually need to know. This course jumps around to a variety of topics that you may be totally unfamiliar with, providing you a small taste of each and then moving on.
```

> The following courses will dive deep into these topics, and you will build dozens of scripts, projects, and websites to cement those skills that will get you hired.

Reviewer: ```suggestion
The following courses will dive deep into these topics. You will build dozens of scripts, projects, and websites to cement those skills that will get you hired.
```
